client/components/pages/WelcomePage/WelcomePage.js

```diff
@@ -1,31 +1,61 @@
-import React, { useEffect } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { push } from 'connected-react-router';
-import R from 'ramda';
-
-import Section from 'react-bulma-companion/lib/Section';
-import Container from 'react-bulma-companion/lib/Container';
-import Title from 'react-bulma-companion/lib/Title';
+import React, { useEffect } from "react";
+// import { useDispatch, useSelector } from "react-redux";
+// import { push } from "connected-react-router";
+// import R from "ramda";
+// import request from "superagent";
+// import Section from "react-bulma-companion/lib/Section";
+// import Container from "react-bulma-companion/lib/Container";
+import Title from "react-bulma-companion/lib/Title";
+import Button from "react-bulma-companion/lib/Button";
 
 export default function WelcomePage() {
-  const dispatch = useDispatch();
-  const { user } = useSelector(R.pick(['user']));
+  // const dispatch = useDispatch();
+  // const { user } = useSelector(R.pick(["user"]));
+  // const [response, setResponse] = React.useState("");
+  // useEffect(() => {
+  //   if (!R.isEmpty(user)) {
+  //     dispatch(push("/home"));
+  //   }
+  // }, []);
 
-  useEffect(() => {
-    if (!R.isEmpty(user)) {
-      dispatch(push('/home'));
-    }
-  }, []);
+  // const handleAddTodo = () => {
+  //   request
+  //     .get("/api/example/")
+  //     .send()
+  //     .then((result) => {
+  //       console.log(result.body.message);
+  //       setResponse(result.body.message);
+  //     })
+  //     .catch();
+  // };
 
   return (
     <div className="welcome-page page">
-      <Section>
-        <Container>
-          <Title size="1">
-            Welcome Page!
-          </Title>
-        </Container>
-      </Section>
+      <div className="top">
+        <div id="open-pic"></div>
+        <Title id="website-title">יער האקלים</Title>
+        <h2 id="sub-title">שותלים עצים, מפריחים ערים</h2>
+        <Button id="btn-login">התחברות</Button>
+        <a id="sign-in" href="#">
+          הרשמה
+        </a>
+      </div>
+      <Title id="welcome" size="4">
```

Comment:
I love it that you used Bulma's title element

Reply:
:)